package.json

```diff
@@ -4,16 +4,16 @@
   "version": "1.0.0",
   "description": "Convert GraphQL Schema to TypeScript defs",
   "dependencies": {
-    "graphql": "^0.10.5"
+    "graphql": "^0.11.7"
```

Comment:
Could this not include both 0.10 and 0.11? Otherwise it's a breaking change

Reply:
yeah let me do a range, actually